src/main/java/gov/nih/nci/evs/api/fhir/R5/CodeSystemProviderR5.java

```diff
@@ -385,12 +401,24 @@ public Parameters validateCodeImplicit(
     }
     try {
       FhirUtilityR5.mutuallyRequired(display, "display", code, "code");
-      FhirUtilityR5.notSupported(codeableConcept, "codeableConcept");
-      FhirUtilityR5.notSupported(codeSystem, "codeSystem");
-      FhirUtilityR5.notSupported(coding, "coding");
-      FhirUtilityR5.notSupported(date, "date");
-      FhirUtilityR5.notSupported(abstractt, "abstract");
-      FhirUtilityR5.notSupported(displayLanguage, "displayLanguage");
+      //      FhirUtilityR5.notSupported(codeableConcept, "codeableConcept");
+      //      FhirUtilityR5.notSupported(codeSystem, "codeSystem");
+      //      FhirUtilityR5.notSupported(coding, "coding");
+      //      FhirUtilityR5.notSupported(date, "date");
+      //      FhirUtilityR5.notSupported(abstractt, "abstract");
+      //      FhirUtilityR5.notSupported(displayLanguage, "displayLanguage");
+      for (final String param :
+          new String[] {
+            "codeableConcept", "codeSystem", "coding", "date", "abstract", "displayLanguage"
+          }) {
+        FhirUtilityR5.notSupported(request, param);
+      }
```

Comment:
Deborah, some of these parameters like "codeableConcept" are not query parameters, they would only be used on a POST request, which we are already blocking.   It's technically OK to have these here but in practice they would never actually get triggered because they're not "named" properties in the request in the way the non-object ones are.  I'd recommend getting rid of "codableConcept" here (which is an object).  The things that are date/string/boolean/integer/or Coding are not post parameters, likely anything else is (unless a primivitve type I missed)